fe/fe-core/src/main/java/com/starrocks/journal/Journal.java

```diff
@@ -57,4 +59,9 @@ public interface Journal {
     // Get all the dbs' name
     public List<Long> getDatabaseNames();
 
+    // Write a jounal within current transaction
+    public void writeWithinTxn(short op, DataOutputBuffer buffer);
```

Comment:
This function is very confusing.
Can this journal support multiple transaction. And if write is called after this function, what will happen?
I think we need to make this clearer.